### PR TITLE
Docs for "reconfig" in ClickHouse Keeper

### PR DESCRIPTION
**Pull request by myrrc**
Depends on https://github.com/ClickHouse/ClickHouse/pull/49450/.

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
This is an automated comment for commit e2fc2f1d369de3e5c3d80cc13ed50a7c618f29fb with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1164/e2fc2f1d369de3e5c3d80cc13ed50a7c618f29fb/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

**Comment by myrrc** [comment]
Hi @DanRoscigno. Why did you make this PR a draft?
I believe the docs for this PR are ready

**Comment by DanRoscigno** [comment]
> Hi @DanRoscigno. Why did you make this PR a draft? I believe the docs for this PR are ready

Hi @myrrc I marked it as a draft to make sure that it does not get merged until the PRs it depends on are merged.

File: docs/en/guides/sre/keeper/index.md [hunk 1]
```diff
@@ -979,3 +981,71 @@ Query id: b047d459-a1d2-4016-bcf9-3e97e30e49c2
...
+  (each part of `joining`, each part of `leaving`) must be decided on separately. Thus there is no bulk
+  reconfiguration available as it would be misleading for end users.
+
+  Changing server type (participant/learner) isn't possible either as it's not supported by NuRaft, and
+  the only way would be to remove and add server, which again would be misleading.
+
+- `from_version` field is not used. All request with set `from_version` are declined.
+- Unlike ZooKeeper, there is no way to wait on cluster reconfiguration by submitting a `sync` command.
+  New config will be _eventually_ applied but with no time guarantees.
+- `reconfig` command may fail for various reasons. You can check cluster's state and see whether the update
```
Comment (antonio2368) [inline docs/en/guides/sre/keeper/index.md:1049]: mention ways how to check cluster configuration (using 4LW or GET on a specific node)

**Review by antonio2368 — APPROVED** [review]

**Pull request merged**